worker/upgradeseries/worker.go

```diff
@@ -335,26 +356,22 @@ func (w *upgradeSeriesWorker) handleCompleted() error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	w.machineStatus = model.UpgradeSeriesNotStarted
+	w.preparedUnits = nil
+	w.completedUnits = nil
+
```

Comment:
Should the above lines go in the `FinishUpgradeSeries` handler? Would it be best to remove the remote state before resetting the local state (similar to the way the uniter does it)?